lib/main.js

```diff
@@ -38,7 +42,18 @@ export default {
       lintOnFly: true,
       lint: (activeEditor) => {
         const command = atom.config.get("linter-ruby.rubyExecutablePath");
+        const ignored = atom.config.get("linter-ruby.ignoredExtensions");
         const filePath = activeEditor.getPath();
+        var ignore = false;
```

Comment:
`const fileExtension = Path.extname(filePath).substr(1)`
[documentation for Path.extname](https://iojs.org/api/path.html#path_path_extname_p)
Also, you should do `const Path = require('path')` where the helpers are defined.